pandas/tests/frame/test_unary.py

```diff
@@ -101,14 +101,16 @@ def test_pos_numeric(self, df):
             # numpy changing behavior in the future
             pytest.param(
                 pd.DataFrame({"a": ["a", "b"]}),
-                marks=[pytest.mark.filterwarnings("ignore")],
+                marks=[
+                    pytest.mark.filterwarnings("ignore:Applying:DeprecationWarning")
```

Comment:
comment on where this comes from and maybe condition under which we can stop filtering? (im guessing numpy>x.y.z)

Reply:
IIUC I don't think there's necessarily a version timeline of when deprecation warnings are enforced (unless that's in scope of numpy 2.0?) cc @seberg

Reply:
Ummm, not sure I want to tie cleaning out all deprecations to NumPy 2.0, but I guess it is prolly safe.  If it helps, you can also put 1.25 and I will just do it, heh...

(We need to clean up expired deprecations a bit more systematically...)

Reply:
Okay I'll just note that it will be changing in 1.25 so we can maybe be more proactive in changing this behavior

Reply:
Just did it for 1.25/main, thanks.